SW2URDF/UI/AssemblyExportForm.cs

```diff
@@ -307,7 +307,10 @@ private void FinishExport(bool exportSTL)
                 Exporter.PackageName = Path.GetFileName(saveFileDialog1.FileName);
 
                 logger.Info("Saving URDF package to " + saveFileDialog1.FileName);
-                Exporter.ExportRobot(exportSTL);
+
+                bool isSTL = radioButtonStl.Checked;
+                Exporter.ExportRobot(exportSTL, isSTL);
```

Comment:
Can you make the mesh format an enum instead of a bool? This will help make it easier to add support for more file formats in the future